### DISCUSSION TITLE
[Feature Request] Support for webp encoding progress monitor

### DISCUSSION THREAD

**Discussion by Cyl18** [ImageMagick/ImageMagick#2871] [Development]
`-monitor` flag for WebP Encode

WebP does not support encoding progress
![image](https://user-images.githubusercontent.com/14993992/100353755-f957b800-3029-11eb-941c-377e6af7d5d4.png)
(png does)
![image](https://user-images.githubusercontent.com/14993992/100353835-1be9d100-302a-11eb-92cc-0e9be7ffd42c.png)

**Comment by urban-warrior** [comment]
Thanks for the problem report.  We can reproduce it and will have a patch to fix it in the GIT master branch @ https://github.com/ImageMagick/ImageMagick later today.  The patch will be available in the beta releases of ImageMagick @ https://www.imagemagick.org/download/beta/ by sometime tomorrow.
